habitat-baselines/habitat_baselines/config/hab3_single_robot.yaml

```diff
@@ -0,0 +1,43 @@
+# @package _global_
+
+defaults:
```

Comment:
Why do we need to create a new config file? I would rather not create new one because we will need to make sure this config is in sync with the other configs. Can we include all these options in the launch files instead?

Reply:
I like using config files for experiments, though maybe I would put them in an experiment folder. It helps us check that we are running things with the same parameters.